pkgs/tools/networking/sslsplit/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, openssl, libevent, libpcap, libnet, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sslsplit";
+  name = "${pname}-${version}";
```

Comment:
`name` is already defined to be exactly this so if you set `pname` you don't need to set `name`.